default/stringtables/en.txt

```diff
@@ -5064,8 +5067,8 @@ Research
 METER_RESEARCH_VALUE_DESC
 '''The Research (or RP "Research Points") of all planets in the empire is combined to discover new [[encyclopedia ENC_TECH]].
 
-Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
-The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
+Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RP, and had a minimum time of 3 turns, 5 RP is the maximum that could be put towards its completion per turn.
+The tech at the top of the queue is given RP first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top. If any RP are left over after giving to all techs in the queue, they are automatically allocated to the tech(s) with the fewest remaining RP to completion.
```

Comment:
You can verb objects -> Objects can be verbed

Also clarify that techs must be unlocked to receive RP.